pkg/operator/starter.go

```diff
@@ -121,38 +127,47 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		),
 	).WithCSIDriverNodeService(
 		"VMwareVSphereDriverNodeServiceController",
-		generated.MustAsset,
+		assets.ReadFile,
 		"node.yaml",
 		kubeClient,
 		kubeInformersForNamespaces.InformersFor(defaultNamespace),
+		nil,
 		WithLogLevelDaemonSetHook(),
 		csidrivernodeservicecontroller.WithObservedProxyDaemonSetHook(),
 	).WithServiceMonitorController(
 		"VMWareVSphereDriverServiceMonitorController",
 		dynamicClient,
-		generated.Asset,
+		assets.ReadFile,
 		"servicemonitor.yaml",
-	).WithExtraInformers(secretInformer.Informer())
+	)
 
 	if err != nil {
 		return err
 	}
 
+	cloudConfigBytes, err := assets.ReadFile("vsphere_cloud_config.yaml")
+	if err != nil {
+		return err
+	}
 	targetConfigController := targetconfigcontroller.NewTargetConfigController(
 		"VMwareVSphereDriverTargetConfigController",
 		defaultNamespace,
-		generated.MustAsset("vsphere_cloud_config.yaml"),
+		cloudConfigBytes,
 		kubeClient,
 		kubeInformersForNamespaces,
 		operatorClient,
 		configInformers,
 		controllerConfig.EventRecorder,
 	)
 
+	scBytes, err :=assets.ReadFile("storageclass.yaml")
```

Comment:
Nit: missing space.

Reply:
This should have been caught by make verify!

Reply:
It is caught by `make verify`, we just don't run it in CI :-)
Fixed.